keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -186,9 +186,11 @@ through a collection of small documents partitioned by group-version.
 All clients of Kubernetes APIs must send a request to every
 group-version in order to "discover" the available APIs. This causes a
 storm of requests for clusters and is a source of latency and
-throttling. This KEP proposes centralizing the "discovery" mechanism
-into one document so that clients only need to make one request to the
-API server to retrieve all the operations available.
+throttling. When new types are added to the API, types will need to be
+fetched again and adds an additional storm of requests. This KEP
+proposes centralizing the "discovery" mechanism into one document so
```

Comment:
There's `/api` and `/apis`, so that's actually two documents? (and two requests, below).